src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs

```diff
@@ -73,5 +81,106 @@ public void AddBuildingBlocksToModule(Module module)
                .Run(_interactionTaskContext.Context));
          }
       }
+
+      public void LoadBuildingBlocksToModule(Module module)
+      {
+         loadBuildingBlocksToModuleBase(module, AskForPKMLFileToOpen);
+      }
+
+      public void LoadBuildingBlocksFromTemplateToModule(Module module)
+      {
+         loadBuildingBlocksToModuleBase(module, openTemplateFile);
+      }
+
+      public void RemoveModule(Module module)
+      {
+         _interactionTaskContext.Context.AddToHistory(new RemoveModuleCommand(module)
+            .Run(_interactionTaskContext.Context));
+      }
+
+      public void AddCloneToProject(Module moduleToClone)
+      {
+         var clonedModule = _interactionTaskContext.Context.Clone(moduleToClone);
```

Comment:
I think the easiest way is to clone the whole module and remove what the user does not want